development/slider-carousel-app/pages/slider-carousel/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+HEAD OBJECT
+title: "ReuseJS slider-carousel"
+author: "Md Amir Sohail"
+---
+import SliderWithButton from "../../components/slider-carousel/Example/SliderWithButton/SliderWithButton.tsx"
+import SimpleSlider from "../../components/slider-carousel/Example/SimpleSlider/SimpleSlider.tsx"
+import SliderWithLinearSlide from "../../components/slider-carousel/Example/SliderWithLinearSlide/SliderWithLinearSlide.tsx"
+
+## Table of contents
+
+## ReuseSliderCarousel Component
+The `ReuseSliderCarousel` component is a reusable carousel component that displays a collection of slides. It provides functionality for `automatic slide transitions`, `navigation buttons`, and `customizable styles`.
+
+- a. React component that renders a slider carousel using the `ReuseSliderCarousel` component from the `@locoworks/reusejs-react-slider-carousel` library. The slider displays a list of slides, and it automatically transitions between slides at a specified interval.
```

Comment:
Is this bullet list or lettered list? Use one or the other, not both.

Reply:
This s Bullet list i removed Letters from all the places where bullet list is used.